tools/pytorch2onnx.py

```diff
@@ -76,11 +76,15 @@ def _demo_mm_inputs(input_shape, num_classes):
 def _prepare_input_img(img_path,
                        test_pipeline,
                        shape=None,
-                       rescale_shape=None):
+                       rescale_shape=None,
+                       normalize_in_graph=False):
     # build the data pipeline
     if shape is not None:
         test_pipeline[1]['img_scale'] = (shape[1], shape[0])
     test_pipeline[1]['transforms'][0]['keep_ratio'] = False
+    for transform in test_pipeline[1]['transforms']:
+        if transform['type'] == 'Normalize':
+            transform.normalize_in_graph = normalize_in_graph
```

Comment:
Should explicitly convert input tensor from `uint8` to `float32`
by update line 93:
```
imgs = [img.float() for img in data['img']]
```